### PR TITLE
Misc/spike flysystem

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: assets/js/file-manager.js [hunk 1]
```diff
@@ -37,10 +37,10 @@ if (fileManagerElement.dataset.id) {
     });
 
     let myDropzone = new Dropzone("div#dropzone-uploader", {
-        url: Routing.generate('pelagos_api_post_files_dataset_submission') + "/" + datasetSubmissionId,
+        url: Routing.generate('pelagos_api_post_chunks'),
         chunking: true,
         chunkSize: 1024*1024,
```
Comment (praneethpr) [inline assets/js/file-manager.js:42]: Change it to 10000000

File: src/Controller/Api/FilesController.php [hunk 2]
```diff
@@ -61,32 +61,28 @@ public function getFiles(int $id)
     }
```
Comment (mickel1138) [inline src/Controller/Api/FilesController.php:61]: Maybe this should be the uploadController?

**Review by mickel1138 — APPROVED** [review]
Looks good, flysystem works.

File: src/Util/FileUploader.php [hunk 3]
```diff
@@ -2,96 +2,78 @@
...
+
+    /**
+     * Relative path for files folder.
+     */
+    const FILES_DIRECTORY = 'upload/files';
+
+    /**
+     * FileUploader constructor.
+     *
+     * @param string              $homedirPrefix    Home directory path prefix.
```
Comment (mickel1138) [inline src/Util/FileUploader.php:38]: Not really using these anymore right?

File: src/Controller/Api/FilesController.php (hunk 2 above)
Comment (praneethpr) [reply src/Controller/Api/FilesController.php:61]: Yes, Makes sense to rename it

File: src/Util/FileUploader.php (hunk 3 above)
Comment (praneethpr) [reply src/Util/FileUploader.php:38]: But we need some kind of directory prefix

**Pull request merged**